x/emissions/keeper/inference_synthesis/network_losses_test.go

```diff
@@ -61,144 +61,6 @@ func (s *InferenceSynthesisTestSuite) TestRunningWeightedAvgUpdate() {
 	}
 }
 
-func (s *InferenceSynthesisTestSuite) TestCalcCombinedNetworkLossSimpleCaseWithOneReputer() {
```

Comment:
is there a reason we're deleting this test, don't love deleting tests without a reason

Reply:
Same, does this test break with the changes?

Reply:
this is a test for the `CalcCombinedNetworkLoss` which is a function that I deleted